### PR TITLE
A few doc fixes + improvements 

### PR DESCRIPTION
**Pull request by aarron-lee**
A few things:

- add more emphasis to the separate EFI partition for dual boot setups
- add ayaneo wifi workaround
- add `bazzite-rollback-helper` example

### CODE REVIEW

File: src/Handheld_and_HTPC_edition/Handheld_Wiki/Ayaneo_Handhelds.md [hunk 1]
```diff
@@ -34,8 +34,11 @@ Read the [**Installing Bazzite on Handheld PCs documentation**](/General/Install
   - Alternatively, since ALC269VB is an audio chipset that has been around for years, it just need its pins to be rerouted with `hdajackretask` (part of fedora's alsa-tools) package. (You basically need to reroute the auto-generated "green headset" and "black headset" devices to "Channel 30 & 31").
   - External audio works over both HDMI and eGPU.
 - **Suspend does not work without a workaround**!
-  - see suspend workaround [here](https://universal-blue.discourse.group/t/ayaneo-handheld-compatibility/2417#p-5599-suspend-workaround-5)
+  - see suspend workaround [here](https://docs.bazzite.gg/Handheld_and_HTPC_edition/Handheld_Wiki/Ayaneo_Handhelds/#suspend-workaround)
```
Comment (Zeglius) [inline src/Handheld_and_HTPC_edition/Handheld_Wiki/Ayaneo_Handhelds.md:37]: Urls need to point to markdown files, not the docs.bazzite.gg doman, see https://github.com/KyleGospo/docs.bazzite.gg#1-internal-links
```suggestion
  - see suspend workaround [here](#suspend-workaround)
```

File: src/Handheld_and_HTPC_edition/Handheld_Wiki/Ayaneo_Handhelds.md [hunk 2]
```diff
@@ -34,8 +34,11 @@ Read the [**Installing Bazzite on Handheld PCs documentation**](/General/Install
...
 - VRAM size option is missing from BIOS as it's controlled by AYASPACE application under windows.
+- For certain Ayaneo devices, wifi stops working after suspend-resume cycles
+  - as a workaround, you need to `modprobe -r mt7921e` right before suspend to disable the wifi, then run `modprobe mt7921e` after suspend to re-enable the wifi
+    - You can find an automatic installer for the workaround [here](https://github.com/aarron-lee/gpd-win-tricks/tree/mt7921e_fix/suspend-fix)
 
 ### Functional HHD\*\*:
```
Comment (Zeglius) [inline src/Handheld_and_HTPC_edition/Handheld_Wiki/Ayaneo_Handhelds.md:43]:
```suggestion
### Functional HHD
```

**Review by Zeglius — COMMENTED** [review]
Also you might as well add yourself as author to the metadata

```
 ---
 authors:
   - "@nicknamenamenick"
+  - "@aarron-lee"
 ---
```

**Comment by aarron-lee** [comment]
all requested changes done :+1:

**Pull request merged**